migrations/schema.sql

```diff
@@ -0,0 +1,95 @@
+--
```

Comment:
Let's try to use this package to manage our migrations. It makes life easier.

Here is the link: https://github.com/pressly/goose